src/manage-storage.js

```diff
@@ -46,9 +46,16 @@ function manage_storage(){
 	);
 
 	// check if there are some images in storage
-	var imagesInStorage = Object.keys(localStorage).some(k => {return k.match(/^image#/);});
+	var images_in_storage =(function(){
+		for(var k in localStorage){
+			if(k.match(/^image#/)){
+				return true;
+			}
+		}
+		return false;
+	})();
```

Comment:
Sorry, I meant just the function syntax. You can still use `Object.keys` and `some`, which was much shorter.

Reply:
Oh my bad. Misunderstanding